### DISCUSSION TITLE
"Choose Entity" dropdown in history is too narrow and therefore entities shown are not unique

### DISCUSSION THREAD

**Discussion by redlefloh** [home-assistant/frontend#15799] [Other feature requests] [answered]
The "Choose Entity" dropdown is too narrow and displays only 15 characters of the entity ID which in most cases is not enough to select a unique entity. One example is OpenWeatherMap sensors, I basically need to guess which sensor to choose. 

![entityid in history 0](https://user-images.githubusercontent.com/24863182/224532623-08a1dbb0-2c7c-42cf-b001-a1761c908d49.png)

here is a comparison, when I list all OpenWeather... Sensors in the entities view we get this:

![entityid in history 1](https://user-images.githubusercontent.com/24863182/224532437-58337209-67ab-4aff-b225-c47a756c4400.png)

with only 15 characters in the drop down, you see that the entity Names and entityIDs are not unique:
 
![entityid in history 2](https://user-images.githubusercontent.com/24863182/224532479-719f4fbd-b687-4857-b57c-27af53f79527.png)

with 50 characters width in the drop down, one would see Names and entityIDs such that they are mostly:

![entityid in history 3](https://user-images.githubusercontent.com/24863182/224532518-8d9ede00-6bc8-4d1d-81b2-3bbf2e2b145b.png)

So it would really be super helpful to make the dropdown wider. Thanks!

**Comment by ildar170975** [comment]
Already registered
https://github.com/home-assistant/frontend/issues/14897

**Comment by karwosts** [comment, marked as answer]
Can you confirm you're on a recent HA version? What you have looks like the old version, AFAIK it should be 400px wide now. 

In recent version the box opens underneath the chips, not to the right, which makes me suspect you're either on an old version, or have something cached that needs to be cleared.

**Reply by ildar170975** [reply]
Even when the box is below selectors - it is narrow:
![image](https://user-images.githubusercontent.com/71872483/225198197-c5f3bcaa-0903-46a4-94a0-60abe36dd571.png)

**Reply by redlefloh** [reply]
thanks, afaik I do run the latest stable version of the docker container, just pulled the latest docker image with 
`docker pull ghcr.io/home-assistant/home-assistant:stable`
and the return is:
`stable: Pulling from home-assistant/home-assistant
Digest: sha256:5ee95ed91a2b1649b9921402e3c06d1ffcb6257261c10dc1d61f38b276d90964
Status: Image is up to date for ghcr.io/home-assistant/home-assistant:stable
ghcr.io/home-assistant/home-assistant:stable`

And http://<myipaddress>:<myport>/config/info says:
 
![Screenshot 2023-03-16 at 17 42 59](https://user-images.githubusercontent.com/24863182/225691379-917656f7-f2a8-417c-9f1f-2e634bd74535.png)

Anything I am missing? What you @karwosts see is definitely not what I am seeing...? It is not opening underneath.

**Reply by karwosts** [reply]
I'm not sure about "stable versions", but 2022.11.2 is 4 months stale, If you update to more recent you will get the wider entity picker.

**Reply by ildar170975** [reply]
@redlefloh do not use "ghcr", what else can I say...
Use this - `docker pull homeassistant/home-assistant:stable`
And the version's name "2022.11.2" is obviously saying that it is old.

**Reply by ildar170975** [reply]
> I'm not sure about "stable versions", but 2022.11.2 is 4 months stale, If you update to more recent you will get the wider entity picker.

Unfortunately it is just a bit wider.

**Reply by redlefloh** [reply]
🙈 sorry, my bad,I copied the docker pull ghcr.io/home-assistant/home-assistant:stable from somewhere and never looked at it again and downloaded a few updates already that obviously didn't work.... 
Thanks for opening my eyes... 
![Screenshot 2023-03-16 at 18 27 49](https://user-images.githubusercontent.com/24863182/225703483-1cee6d02-09e0-4cd2-af64-fe91b98f629b.png)
and "widening" my view:
![Screenshot 2023-03-16 at 18 29 56](https://user-images.githubusercontent.com/24863182/225703985-5c43f3bd-3ca3-41f2-a723-374f7580b53e.png)
